pkg/gatherers/clusterconfig/recent_metrics.go

```diff
@@ -77,7 +77,6 @@ func (g *Gatherer) GatherMostRecentMetrics(ctx context.Context) ([]record.Record
 
 func gatherMostRecentMetrics(ctx context.Context, metricsClient rest.Interface) ([]record.Record, []error) {
 	data, err := metricsClient.Get().AbsPath("federate").
-		Param("match[]", "etcd_object_counts").
```

Comment:
It would be probably good to mentioned when it was removed....I will try to find this information